core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3502,53 +3502,53 @@ public String toString() {
+      durationBuilder(Name.MASTER_THROTTLE_ACTIVE_HEAP_GC_TIME)
           .setDefaultValue("1sec")
           .build();
-  public static final PropertyKey MASTER_THROTTLE_NORMAL_RPC_QUEUE_SIZE =
-      intBuilder(Name.MASTER_THROTTLE_NORMAL_RPC_QUEUE_SIZE)
+  public static final PropertyKey MASTER_THROTTLE_ACTIVE_RPC_QUEUE_SIZE =
+      intBuilder(Name.MASTER_THROTTLE_ACTIVE_RPC_QUEUE_SIZE)
           .setDefaultValue(50000)
           .setIsDynamic(true)
           .build();
-  public static final PropertyKey MASTER_THROTTLE_LOAD_CPU_LOAD_RATIO =
-      doubleBuilder(Name.MASTER_THROTTLE_LOAD_CPU_LOAD_RATIO)
+  public static final PropertyKey MASTER_THROTTLE_STRESSED_CPU_LOAD_RATIO =
+      doubleBuilder(Name.MASTER_THROTTLE_STRESSED_CPU_LOAD_RATIO)
           .setDefaultValue(0.8d)
           .build();
-  public static final PropertyKey MASTER_THROTTLE_LOAD_HEAP_USED_RATIO =
-      doubleBuilder(Name.MASTER_THROTTLE_LOAD_HEAP_USED_RATIO)
+  public static final PropertyKey MASTER_THROTTLE_STRESSED_HEAP_USED_RATIO =
+      doubleBuilder(Name.MASTER_THROTTLE_STRESSED_HEAP_USED_RATIO)
           .setDefaultValue(0.8d)
           .build();
-  public static final PropertyKey MASTER_THROTTLE_LOAD_HEAP_GC_TIME =
-      durationBuilder(Name.MASTER_THROTTLE_LOAD_HEAP_GC_TIME)
+  public static final PropertyKey MASTER_THROTTLE_STRESSED_HEAP_GC_TIME =
+      durationBuilder(Name.MASTER_THROTTLE_STRESSED_HEAP_GC_TIME)
           .setDefaultValue("5sec")
           .build();
-  public static final PropertyKey MASTER_THROTTLE_LOAD_RPC_QUEUE_SIZE =
-      intBuilder(Name.MASTER_THROTTLE_LOAD_RPC_QUEUE_SIZE)
+  public static final PropertyKey MASTER_THROTTLE_STRESSED_RPC_QUEUE_SIZE =
+      intBuilder(Name.MASTER_THROTTLE_STRESSED_RPC_QUEUE_SIZE)
           .setDefaultValue(100000)
           .build();
-  public static final PropertyKey MASTER_THROTTLE_STRESS_CPU_LOAD_RATIO =
-      doubleBuilder(Name.MASTER_THROTTLE_STRESS_CPU_LOAD_RATIO)
+  public static final PropertyKey MASTER_THROTTLE_OVERLOADED_CPU_LOAD_RATIO =
+      doubleBuilder(Name.MASTER_THROTTLE_OVERLOADED_CPU_LOAD_RATIO)
           .setDefaultValue(0.95d)
           .build();
-  public static final PropertyKey MASTER_THROTTLE_STRESS_HEAP_USED_RATIO =
-      doubleBuilder(Name.MASTER_THROTTLE_STRESS_HEAP_USED_RATIO)
+  public static final PropertyKey MASTER_THROTTLE_OVERLOADED_HEAP_USED_RATIO =
+      doubleBuilder(Name.MASTER_THROTTLE_OVERLOADED_HEAP_USED_RATIO)
           .setDefaultValue(0.9d)
           .build();
-  public static final PropertyKey MASTER_THROTTLE_STRESS_HEAP_GC_TIME =
-      durationBuilder(Name.MASTER_THROTTLE_STRESS_HEAP_GC_TIME)
+  public static final PropertyKey MASTER_THROTTLE_OVERLOADED_HEAP_GC_TIME =
+      durationBuilder(Name.MASTER_THROTTLE_OVERLOADED_HEAP_GC_TIME)
           .setDefaultValue("10sec")
           .build();
-  public static final PropertyKey MASTER_THROTTLE_STRESS_RPC_QUEUE_SIZE =
-      intBuilder(Name.MASTER_THROTTLE_STRESS_RPC_QUEUE_SIZE)
```

Comment:
Maybe add a validate function for these properties here https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/conf/InstancedConfiguration.java#L368

Reply:
Good call

Reply:
See change: https://github.com/Alluxio/alluxio/pull/16190/files#r990624386